jbmc/src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -1107,10 +1108,11 @@ code_blockt java_string_library_preprocesst::make_init_function_from_call(
   code_blockt code;
 
   // Processing parameters
-  exprt::operandst args=process_parameters(params, loc, symbol_table, code);
+  const exprt::operandst args =
+    process_parameters(params, loc, symbol_table, code);
```

Comment:
Please `std::move` (and thus don't `const`) this, that's an unnecessary copying of a vector.

Reply:
I think the move constructor would already be used here since the right-hand-side is movable, and the function where it's used takes the argument by reference, so there is not point using move there.

Reply:
As far as I understand `args` is not implicitly moved here for two reasons: 1) it's a named object and 2) it's `const`. But then also my comment may have been unclear: the value will be moved into `args`, but I was asking for it also to be moved out of `args`.

Reply:
Ok I thought you wanted to move into args. Moving it out of args will have no effect here because the function `string_expr_of_function` takes a const ref.

Reply:
Ok, sorry, please disregard my comment in that case.